lib/grpnames.gd

```diff
@@ -267,17 +268,56 @@ DeclareGlobalFunction( "DirectFactorsOfGroupKN", IsGroup );
 
 #############################################################################
 ##
-#A  SemidirectFactorsOfGroup( <G> ) . decomposition into a semidirect product
```

Comment:
I was very mildly concerned about the removal (resp. renaming of `SemidirectFactorsOfGroup`). However, it was undocumented, and no package I am aware of uses it. So, in the end I have no objections to that.